virtual-machine/ansible.yaml

```diff
@@ -60,7 +61,11 @@ node_templates:
       ansible_env_vars: &env_vars
         AWS_ACCESS_KEY_ID: { get_secret: aws_access_key_id }
         AWS_SECRET_ACCESS_KEY: {get_secret: aws_secret_access_key }
-        AWS_REGION: { get_input: aws_region_name} 
+        AWS_REGION: { get_input: aws_region_name}
+      extra_packages: &extra_packages
+        - boto3==1.16.26
```

Comment:
may conflict with the version in aws plugin

Reply:
We install it on playbooks venv so i dont think it will make any problems .